plugins/ubiquiti/unifi_api

```diff
@@ -0,0 +1,1166 @@
+#!/usr/bin/perl
+# -*- perl -*-
+
+=encoding utf8
+
+=head1 NAME
+
+unifi_api - Munin plugin to display device and network information from the 
+            Ubiquiti unifi API
+
+=head1 APPLICABLE SYSTEMS
+
+Unifi controllors with direct API access
+
+Controller version 5+ required (tested with 5.8.x)
+
+WebRTC is not supported at this time
+
+=head1 CONFIGURATION
+
+This script uses the multigraph functionality to generate many graphs.
+
+You will need to supply your API login details:
+
+  [unifi_api]
+    env.user Controller_Username
+    # default is ubnt
```

Comment:
I think, it is quite common to put comments *above* the line in question (and not below).
Maybe you want to change this?
(just a matter of taste, I guess)

Reply:
Aye.  also re-worked the section with some sub headings to be a bit easier to read.